src/core/public/chrome/nav_links/to_nav_link.test.ts

```diff
@@ -174,4 +179,38 @@ describe('toNavLink', () => {
       })
     );
   });
+
+  it('uses the workspaceVisibility of the application to construct the url', () => {
```

Comment:
`workspaceVisibility` doesn't match the prop.

Reply:
Fix for that.